### PR TITLE
User Authentication Sign Up

### PR DESCRIPTION
**Pull request by aniyahb**
Video of sign up and login functioning: 

<div>
    <a href="https://www.loom.com/share/d1ee51e73aea4942a62dde9c9d9b764e">
      <p>User Authentication Functionality  - Watch Video</p>
    </a>
    <a href="https://www.loom.com/share/d1ee51e73aea4942a62dde9c9d9b764e">
      
    </a>
  </div>

### CODE REVIEW

File: frontend/src/RecipeList/RecipeList.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    const [modalOpen, setModalOpen] = useState(false);
+    const [pickedRecipe, setPickedRecipe] = useState(null);
+    const [sortBy, setSortBy] = useState('Popular');
+    const [search, setSearch] = useState("");
+    const [url, setUrl] = useState('')
+
+
+
+    useEffect(() => {
+        const getPopular = async () => {
```
Comment (KatsiarynaMashokha) [inline frontend/src/RecipeList/RecipeList.jsx:22]: let's rename this to smth more clear: `getPopularRecipes`?

File: frontend/src/App.css [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+  align-items: center;
+  height:9.5%;
+  border-bottom:10px #171717;
+  box-shadow: 0px 0.5px 5px ;
+
+
+}
+
+
+/* vite default code
```
Comment (KatsiarynaMashokha) [inline frontend/src/App.css:24]: can you pls remove this commented out code?

File: backend/index.js [hunk 3]
```diff
@@ -1,10 +1,49 @@
...
+const prisma = new PrismaClient
 const app = express()
+app.use(express.json())
+
 const PORT = process.env.PORT || 2500
 
-app.use(express.json())
+
+// Sign Up Endpoint
+app.post('/signUp', async (req, res) => {
```
Comment (KatsiarynaMashokha) [inline backend/index.js:13]: could you pls make all urls lowercased?

File: frontend/src/LogIn/LogIn.jsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
+import './LogIn.css'
+import React, { useState, useContext, createContext} from "react";
+import {useNavigate,Link} from "react-router-dom"
+// import  {UserContext}  from '../UserContext'
```
Comment (KatsiarynaMashokha) [inline frontend/src/LogIn/LogIn.jsx:4]: let's remove this

**Comment by alishahimtiaz** [comment]
I think this PR is doing too much, your branches show recipe list, and signup, when we'd only want one branch and have the rest on separate branches

**Pull request merged**